spec/chatbox.js

```diff
@@ -1387,8 +1391,10 @@ describe("Chatboxes", function () {
             _converse.handleMessageStanza(msgFactory());
             await u.waitUntil(() => chatbox.messages.length);
             expect(chatbox.get('num_unread')).toBe(1);
+            expect(chatbox.get('first_unread')).toBeDefined();
             _converse.saveWindowState(null, 'focus');
             expect(chatbox.get('num_unread')).toBe(1);
+            expect(chatbox.get('first_unread')).toBeDefined();
```

Comment:
Please add tests to check that the right message gets marked as the first unread.

Reply:
ok. I will modify to confirm the expected msgid is set in the first_uread_id attribute